tests/component_tests/orc_evaporator_tests.py

```diff
@@ -0,0 +1,129 @@
+                          design=['pr1', 'pr2', 'pr3'],
+                          offdesign=['zeta1', 'zeta2', 'zeta3'])
+        self.c1.set_attr(T=146.6, p=4.34, m=20.4, state='g',
+                         fluid={'water': 1, 'Isopentane': 0})
+        self.c3.set_attr(T=146.6, p=10.2, m=190.8,
+                         fluid={'water': 1, 'Isopentane': 0})
+        self.c4.set_attr(T=118.6)
+        self.c5.set_attr(T=111.6, p=10.8,
+                         fluid={'water': 0, 'Isopentane': 1})
+
+        # test heat transfer
+        Q = -6.64e+07
+        self.c3.set_attr(m=np.nan)
+        instance.set_attr(Q=Q)
+        self.nw.solve('design')
+        self.nw.save('tmp')
+        Q_is = self.c5.m.val_SI * (self.c6.h.val_SI - self.c5.h.val_SI)
+        msg = ('Value of heat flow must be ' + str(round(Q, 0)) +
+               ', is ' + str(round(Q_is, 0)) + '.')
+        eq_(round(Q, 0), round(Q_is, 0), msg)
+
+        # test bus
+        P = 6.64e+07
+        b = bus('heat transfer', P=P)
+        b.add_comps({'c': instance})
+        self.nw.add_busses(b)
+        self.c4.set_attr(T=np.nan)
+        self.nw.solve('design')
+
+        msg = ('Value of heat flow must be ' + str(round(P, 0)) +
+               ', is ' + str(round(instance.Q.val, 0)) + '.')
+        eq_(round(P, 0), round(instance.Q.val, 0), msg)
+
+        # Check the state of the steam and working fluid outlet:
+        x_outl1_calc = self.c2.x.val
+        x_outl3_calc = self.c6.x.val
+        zeta1 = instance.zeta1.val
+        zeta2 = instance.zeta2.val
+        zeta3 = instance.zeta3.val
+
+        msg = ('Vapor mass fraction of steam outlet must be 0.0, is ' +
+               str(round(x_outl1_calc, 1)) + '.')
+        eq_(round(x_outl1_calc, 1), 0.0, msg)
+
+        msg = ('Vapor mass fraction of working fluid outlet must be 1.0, is ' +
+               str(round(x_outl3_calc, 1)) + '.')
+        eq_(round(x_outl3_calc, 1), 1.0, msg)
+
+        # Check offdesign by zeta values
+        # geometry independent friction coefficient
+        self.nw.solve('offdesign', design_path='tmp')
+
+        msg = ('Geometry independent friction coefficient '
+               'at hot side 1 (steam) '
+               'must be ' + str(round(zeta1, 1)) + ', is ' +
+               str(round(instance.zeta1.val, 1)) + '.')
+        eq_(round(instance.zeta1.val, 1), round(zeta1, 1), msg)
+        msg = ('Geometry independent friction coefficient at '
+               'hot side 2 (brine) '
+               'must be ' + str(round(zeta2, 1)) + ', is ' +
+               str(round(instance.zeta2.val, 1)) + '.')
+        eq_(round(instance.zeta2.val, 1), round(zeta2, 1), msg)
+        msg = ('Geometry independent friction coefficient at cold side '
+               '(Isopentane) must be ' + str(round(zeta3, 1)) + ', is ' +
+               str(round(instance.zeta3.val, 1)) + '.')
+        eq_(round(instance.zeta3.val, 1), round(zeta3, 1), msg)
+
+        shutil.rmtree('./tmp', ignore_errors=True)
```

Comment:
There should be tests regarding the subcooling and overheating parameters.

Reply:
Added.